common/modal.js

```diff
@@ -215,12 +216,11 @@
          * In that case, we might have the `matchNotNull` variable. If we do, we just need to pass that.
          */
         function getMultiSelectionResult() {
-            var res = vm.tableModel.selectedRows;
-            if (!Array.isArray(res)) res = [];
             if (vm.tableModel.matchNotNull) {
-                res = {matchNotNull: true};
+                return {matchNotNull: true};
             }
-            return res;
+            var res = vm.tableModel.selectedRows;
+            return Array.isArray(res) ? res : [];
```

Comment:
Returning an array here, but above you return an object..

Reply:
This can be refactored.